CHANGELOG.md

```diff
@@ -7,11 +7,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+<!-- List New Changes Here -->
+
+## [0.7.0] - 2017-01-12
```

Comment:
Keep the changelog versions as is on master, we'll make sure it gets merged correctly when we cut the next release.